cmd/krew/cmd/index.go

```diff
@@ -75,7 +75,13 @@ var indexAddCmd = &cobra.Command{
 		if !indexoperations.IsValidIndexName(name) {
 			return errInvalidIndexName
 		}
-		return indexoperations.AddIndex(paths, name, args[1])
+		err := indexoperations.AddIndex(paths, name, args[1])
+		if err == nil {
```

Comment:
I think `if err != nil { return err}`  then printing this outside the if-block would be better.

also worth adding tests for (just to check WARNING: exists)